clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -308,6 +308,11 @@ Changes in existing checks
   <clang-tidy/checks/misc/const-correctness>` check to avoid false positive when
   using pointer to member function.
 
+- Improved :doc:`misc-const-correctness
```

Comment:
there is already entry for this check, just merge changes.

Reply:
Done. I have also reworded the release note, I think it reads better now.